packages/create-subscription/README.md

```diff
@@ -11,6 +11,16 @@ Other cases have **better long-term solutions**:
 * I/O subscriptions (e.g. notifications) that update infrequently should use [`simple-cache-provider`](https://github.com/facebook/react/blob/master/packages/simple-cache-provider/README.md) instead.
 * Complex libraries like Relay/Apollo should manage subscriptions manually with the same techniques which this library uses under the hood (as referenced [here](https://gist.github.com/bvaughn/d569177d70b50b58bff69c3c4a5353f3)) in a way that is most optimized for their library usage.
 
+## Compatibility with async mode
```

Comment:
change to "Incompatibilities with async mode"? :p

Reply:
Maybe "limitations"? To communicate that it's correct, but not ideally scheduled.